results-processor/wptreport.py

```diff
@@ -515,11 +515,13 @@ def _channel_to_labels(browser: str, channel: str) -> Set[str]:
     if channel == 'dev' or channel == 'preview':
         # e.g. Chrome Dev and Safari Technology Preview
         labels.add('experimental')
-    if channel == 'nightly' and browser == 'firefox':
-        # Only treat Firefox Nightly, but not Chrome Nightly, as experimental.
+    if channel == 'nightly' and \
+            browser in ('firefox', 'webkitgtk_minibrowser'):
```

Comment:
Should we just check `!= chrome` then, to allow future browsers to meet this criteria?

Reply:
```suggestion
            browser != 'chrome':
```

Yeah that sounds like a better idea.